conbench/templates/compare-entity.html

```diff
@@ -33,17 +33,10 @@
   <nav aria-label="breadcrumb">
     <ol class="breadcrumb">
       <li class="breadcrumb-item active">Compare</li>
-      <li class="breadcrumb-item active">
-        <a href="{{ compare_runs_url }}">Runs</a>
-      </li>
-      <li class="breadcrumb-item active">
-        <a href="{{ compare_batches_url }}">Batches</a>
-      </li>
-      <li class="breadcrumb-item active">Benchmarks</li>
       <li class="breadcrumb-item active" aria-current="page">
-        <a href="{{ url_for('app.benchmark-result', benchmark_result_id=baseline_id) }}">Baseline</a>
+        <a href="{{ url_for('app.benchmark-result', benchmark_result_id=baseline_id) }}">baseline result</a>
         ...
-        <a href="{{ url_for('app.benchmark-result', benchmark_result_id=contender_id) }}">Contender</a>
+        <a href="{{ url_for('app.benchmark-result', benchmark_result_id=contender_id) }}">reference result</a>
```

Comment:
Is this a purposeful vocabulary change you want to make across the webapp, API, and `benchalerts`? We've been fairly strict on using "contender" for a while now, and I want to be sure we remain consistent. Personally I think "reference" is less clear, because I'd guess that a reference result == the "control" result, but I agree that "contender" isn't perfect either.

Reply:
Agreed, at a glance I'd associate "reference" more with "baseline" than "contender".

Maybe we could move more towards older vs newer? Right now you could flip those, but we should warn if you do, since it will almost always be a mistake

Reply:
> I'd associate "reference" more with "baseline" than "contender".

Yes! This was a typo. Elsewhere I had sometimes used `baseline (reference)`, i.e. we agree that if at all, then "reference" is conceptually close to "baseline", and kind of the inverse of "contender".

I think in this particular view I was thinking that when comparing just two results then the presentation should almost be neutral as in A vs. B without assigning specific properties / bias, and I was also thinking that maybe what matters is "old" vs. "new".

Backing out of this though in this PR, let's just keep things for today.

Reply:
took this to https://github.com/conbench/conbench/issues/1369